apps/consent/app/components/captcha-challenge/index.tsx

```diff
@@ -11,11 +17,22 @@ const CaptchaChallengeComponent: React.FC<{
     login_challenge: string
     phone: string
     remember: string
+    channel: string
   }
 }> = ({ id, challenge, formData }) => {
+  const sendMockData = async () => {
+    const mockCaptchaResult = {
+      geetest_challenge: "mockChallenge",
+      geetest_validate: "mockValidate",
+      geetest_seccode: "mockSecCode",
+    }
+    const res = await sendPhoneCode(mockCaptchaResult, formData)
```

Comment:
maybe I'm not understanding the code correctly, but there is no need to fetch the sendPhoneCode endpoint in dev mode, because no code is been sent. 

the action should be as simple as moving to the next screen instead.

Reply:
In this, we are just sending mock data. I thought this would mimic production mode better, but yes, we can just redirect.